src/GitTfs/Core/GitTfsRemote.cs

```diff
@@ -41,12 +41,26 @@ public class GitTfsRemote : IGitTfsRemote
             Aliases = (info.Aliases ?? Enumerable.Empty<string>()).ToArray();
             IgnoreRegexExpression = info.IgnoreRegex;
             IgnoreExceptRegexExpression = info.IgnoreExceptRegex;
+            GitIgnorePath = _remoteOptions.GitIgnorePath ?? info.GitIgnorePath;
+            UseGitIgnore = !_remoteOptions.NoGitIgnore && (_remoteOptions.UseGitIgnore || IsGitIgnoreSupportEnabled());
 
             Autotag = info.Autotag;
 
             IsSubtree = CheckSubtree();
         }
 
+        private bool IsGitIgnoreSupportEnabled()
+        {
+            var isGitIgnoreSupportDisabled = true;
+
+            var value = Repository.GetConfig<string>(GitTfsConstants.DisableGitignoreSupport, null);
```

Comment:
I'm not sure but I think we've got a method to read book config. To verify...

Reply:
Possibly, but this part is just extracted as-is from elsewhere (same file, some lines above, `GitTfsRemote` constructor, see deletion at https://github.com/git-tfs/git-tfs/pull/1214/commits/d7833c489511881c00c622e9e841a1c31865ed67#diff-1c5db4420736d0b07a3d0b90670d93cdL60), so no changes in regards to how configuration was read before it (not saying it can't be further improved, if preferred, but just that this commit didn't make the situation any worse, at least).